backend/schema/corpora_schema.md

```diff
@@ -1,125 +1,71 @@
-where we can't evolve schema and curation policies because it requires expensive software updates. There will be situations where software features or
-capabilities will be hard or impossible because we prioritize this flexibility.
+Then, when the submitter uploads the dataset to the Data Portal, the Data Portal will verify that the dataset does indeed follow an accepted version of the
+schema. If it does not, it will reject the dataset with an appropriate error message.
 
 ## Schema
 
-### Original Dataset Requirements
-
-
-These are the fundamental requirements that all dataset must meet, the handful of things our software depends on and mandatory legal requirements. All
-submitted data must comply with these requirements.
+### Basic Requirements
 
+There are a few requirements that are needed for the cellxgene Explorer to work correctly or are mandatory legal requirements:
 
 *   **Unique observation identifiers**. Each observation (usually a cell) must have an id that is unique within the dataset. This is usually already
     present in every submission, for example as a barcode.
 *   **Unique feature identifiers**. Every feature (usually a gene or transcript) also needs a unique identifier. This is occasionally not present because
-    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will be summed during
-    curation.
-*   **Type annotations or inferrable types**. For both expression values and metadata, we don't want to have to guess at the type. Typing is already explicit
-    in anndata, loom, Seurat and Bioconductor formats.
+    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will need to be appropriately
+    combined before submission. For example, raw counts will be summed and logged counts will be exponentiated, summed, and logged.
 *   **No PII**. No metadata can be personally identifiable: no names, dates of birth, specific locations, etc. There's a
     [list](https://docs.google.com/document/d/1nlUuRiQ8Awo_QsiVFi8OWdhGONjeaXE32z5GblVCfcI/edit?usp=sharing).
 
-
-#### Matrix Layers
+### Matrix Layers
 
 The count matrix itself can exist in several forms. Generally, there is a "raw" count matrix without scaling, filtering, normalization, etc. And there is
 a "final" matrix that is used in the publication with QC and various corrections. There can also be several intermediate matrices.
 
-These different transformations of the count matrix are called "layers" in several file formats. Submissions to Corpora must contain at least one layer,
-the "raw" layer. They may optionally also have a "final" layer used for presentation, and each layer must be identified.
-
-
-### Remix Metadata
+These different transformations of the count matrix are called "layers" in several file formats. Submissions to the Data Portal must contain the "raw"
+layer.
```

Comment:
If the contributor does not have a preference as to which "final" layer to display, does cellxgene have a preference? For example, normalized vs integrated/batched corrected by Seurat.If the contributor does not have a preference as to which "final" layer to display, does cellxgene have a preference? For example, normalized vs integrated/batched corrected by Seurat.

Reply:
cellxgene just displays whatever is written to the `.X` matrix.